src/Bicep.Core.UnitTests/Registry/OciModuleRegistryTests.cs

```diff
@@ -0,0 +1,209 @@
+      ""size"": 735131,
+      ""annotations"": {}
+    }
+  ]
+}";
+            (OciModuleRegistry ociModuleRegistry, OciArtifactModuleReference ociArtifactModuleReference) = GetOciModuleRegistryAndOciArtifactModuleReference(
+                "output myOutput string = 'hello!'",
+                manifestFileContents,
+                "test.azurecr.io",
+                "bicep/modules/storage",
+                "sha:12345");
+
+            ociModuleRegistry.TryGetDocumentationUrl(ociArtifactModuleReference, out _).Should().BeFalse();
+        }
+
+        [DataRow("")]
+        [DataRow("   ")]
+        [DataTestMethod]
+        public void TryGetDocumentationUrl_WithAnnotationsInManifestFileAndInvalidDocumentationUrl_ShouldReturnFalse(string documentationUrl)
+        {
+            var manifestFileContents = @"{
+  ""schemaVersion"": 2,
+  ""artifactType"": ""application/vnd.ms.bicep.module.artifact"",
+  ""config"": {
+    ""mediaType"": ""application/vnd.ms.bicep.module.config.v1+json"",
+    ""digest"": ""sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"",
+    ""size"": 0,
+    ""annotations"": {}
+  },
+  ""layers"": [
+    {
+      ""mediaType"": ""application/vnd.ms.bicep.module.layer.v1+json"",
+      ""digest"": ""sha256:9846dcfde47a4b2943be478754d1169ece3adc6447c9596d9ba48e2579c24173"",
+      ""size"": 735131,
+      ""annotations"": {}
+    }
+  ],
+  ""annotations"": {
+    ""documentation"": """+ documentationUrl + @"""
+  }
+}";
+            (OciModuleRegistry ociModuleRegistry, OciArtifactModuleReference ociArtifactModuleReference) = GetOciModuleRegistryAndOciArtifactModuleReference(
+                "output myOutput string = 'hello!'",
+                manifestFileContents,
+                "test.azurecr.io",
+                "bicep/modules/storage",
+                "sha:12345");
+
+            ociModuleRegistry.TryGetDocumentationUrl(ociArtifactModuleReference, out _).Should().BeFalse();
+        }
+
+        [TestMethod]
+        public void TryGetDocumentationUrl_WithValidDocumentationUrlInManifestFile_ShouldReturnTrue()
+        {
+            var documentationUrl = @"https://github.com/Azure/bicep-registry-modules/blob/main/modules/samples/hello-world/README.md";
+            var manifestFileContents = @"{
+  ""schemaVersion"": 2,
+  ""artifactType"": ""application/vnd.ms.bicep.module.artifact"",
+  ""config"": {
+    ""mediaType"": ""application/vnd.ms.bicep.module.config.v1+json"",
+    ""digest"": ""sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"",
+    ""size"": 0,
+    ""annotations"": {}
+  },
+  ""layers"": [
+    {
+      ""mediaType"": ""application/vnd.ms.bicep.module.layer.v1+json"",
+      ""digest"": ""sha256:9846dcfde47a4b2943be478754d1169ece3adc6447c9596d9ba48e2579c24173"",
+      ""size"": 735131,
+      ""annotations"": {}
+    }
+  ],
+  ""annotations"": {
+    ""documentation"": """+ documentationUrl + @"""
```

Comment:
>documentation [](http://example.com/codeflow?start=6&length=13)

How did this serialize as "documentation" rather than "documentationUrl"?

Reply:
Ah I updated the code after I sent the PR, but forgot to update tests. My bad. Updated it here:
https://github.com/Azure/bicep/commit/c25f355b150acec42d65e75f28d6ae17321eb63f

Thanks!